### ISSUE TITLE
The generated video flickers a bit

### ISSUE THREAD

**Issue by Song367** [fudan-generative-vision/hallo#69]

**Comment by AricGamma** [comment]
Could you upload your cases in this issue? Thanks ❤️

**Comment by 2502128021** [comment]
I agree, the flicker can be observed in many cases, it looks like gaussian noise.

https://github.com/fudan-generative-vision/hallo/assets/27936263/38a5e503-65da-4fe8-ba5d-152fafe7ff07

**Comment by Song367** [comment]
> 我同意，在很多情况下可以观察到闪烁，它看起来像高斯噪声。
> 
>  输出05.mp4

对的，和这个有点差不多

**Comment by yongsuH** [comment]
Did you find a way to resolve this?
